systemtests/src/test/java/io/enmasse/systemtest/common/api/ApiServerTest.java

```diff
@@ -53,7 +53,7 @@
 import static org.junit.jupiter.api.Assertions.*;
 
 @Tag(isolated)
-class ApiServerTest extends TestBase {
+public class ApiServerTest extends TestBase {
```

Comment:
why you changed that to public? in junit5 you dont need to have test class and test method public

Reply:
to make accessible method `simpleMQTTSendReceive`

Reply:
makes sense